### PR TITLE
remove overflow scrollbar(fixes 4055)

### PR DESCRIPTION
**Pull request by ScottHuangNYU**

### CODE REVIEW

File: src/app/shared/forms/planet-tag-input-dialog.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <span mat-dialog-title i18n *ngSwitchCase="'filter'">Collections</span>
   <span mat-dialog-title i18n *ngSwitchCase="'add'">Change Collections</span>
 </ng-container>
-<mat-dialog-content>
+<mat-dialog-content style= "max-height: calc(100vh - 310px)">
```
Comment (lmmrssa) [inline src/app/shared/forms/planet-tag-input-dialog.component.html]: @ScottHuangNYU we do not want to implement inline styling. Can you move this to style file.

Comment (ScottHuangNYU) [reply src/app/shared/forms/planet-tag-input-dialog.component.html]: @lmmrssa I am working on it.

**Comment by ScottHuangNYU** [comment]
@paulbert  Can you explain this more specific？ I can't find a container or a div which represents the whole dialog in PlanetTagInputComponent so I don't know how to set the height of `mat-dialog-content` based on the total height for the dialog.

**Comment by ScottHuangNYU** [comment]
@paulbert After change the padding of `mat-dialog-actions`, it still have two overlap scrollbars.

**Pull request merged**